gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/v4/DefaultApiReactorFactory.java

```diff
@@ -90,7 +99,8 @@ public AsyncReactorFactory(
         final ApiProcessorChainFactory apiProcessorChainFactory,
         final ApiMessageProcessorChainFactory apiMessageProcessorChainFactory,
         final io.gravitee.gateway.jupiter.handlers.api.flow.resolver.FlowResolverFactory flowResolverFactory,
-        final FlowResolverFactory v4FlowResolverFactory
+        final FlowResolverFactory v4FlowResolverFactory,
+        final HttpRequestTimeoutConfiguration httpRequestTimeoutConfiguration
```

Comment:
We should consider use a "reactor timeout" settings instead of http based one.

Reply:
I rename the class to `RequestTimeoutConfiguration` to make it more generic 👍